docs/pre-reqs.md

```diff
@@ -12,7 +12,7 @@ Details on installation and configuration for the AWS CLI can be found [here](ht
 - [ ] This step requires some manual intervention to approve a connection once you create it with the following command:
 ```bash
 aws codestar-connections create-connection \
-    --provider-type Github \
+    --provider-type GitHub \
```

Comment:
Ouch, and nice find!  I wonder if this is new...seems like I would have run into this if it was a problem.  

Thanks for the PR!